frontend/__tests__/module/k8s/k8s-models.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('referenceFor', () => {
 
 describe('referenceForCRD', () => {
   it('returns a reference for custom resource definitions', () => {
-    expect(referenceForCRD(testCRD)).toEqual('testapp.coreos.com~v1alpha1~TestResource');
+    expect(referenceForCRD(testCRD)).toEqual('testapp.coreos.com~v1~TestResource');
```

Comment:
If you revert the change to `testCRD.spec.versions[0].name`, you can also revert this change:
```suggestion
    expect(referenceForCRD(testCRD)).toEqual('testapp.coreos.com~v1alpha1~TestResource');
```